docs/pipeline_config_documentation/pipeline_ingestion_yml.md

```diff
@@ -0,0 +1,494 @@
+ 
+
+### Doublets on RNA - Scrublet
+
+* <p class="parameter">scr</p><br>
+    
+    Specify the following parameters for doublet detection using the tool Scrublet.
+
+    * <p class="parameter">expected_doublet_rate</p> Float, Default: 0.06
+        
+        Fraction of observations (cells) expected to be doublets.
+        Note that results are not particularly sensitive to this parameter. 
+  
+    * <p class="parameter">sim_doublet_ratio</p> Float, Default: 2
+        
+        Number of doublets to simulate, relative to the number of total observations.
+        Note that setting this too high drastically increases computationally complexity.
+        The minimum value we tested is 0.5.
+      
+    * <p class="parameter">n_neighbours</p>Integer, Default: 20
+        
+        Number of neighbors used to construct the k-nearest-neighbor (KNN) classifier for the observations and simulated doublets.
+        The value (round(0.5*sqrt(n_cells))) usually works well.
+    
+    * <p class="parameter">min_counts</p>Integer, Default: 2
+        
+        Used for gene filtering prior to PCA.
+        Genes expressed less than `min_counts` times in `min_cells` cells (see next parameter) are excluded from the data for further processing.
+
+    * <p class="parameter">min_cells</p>Integer, Default: 3
+      
+        Used for gene filtering prior to PCA.
+        Genes expressed less than `min_counts` (see previous parameter) times in `min_cells` cells are excluded from the data for further processing."
+  
+    * <p class="parameter">min_gene_variability_pctl</p>Integer, Default: 85
+      
+        Used for gene filtering prior to PCA.
+        Keeps only the highly variable genes in the top min_gene_variability_pctl percentile for downstream processing, as measured by the v-statistic [Klein et al., Cell 2015].
+  
+    * <p class="parameter">n_prin_comps</p> Integer, Default: 30
+      
+        Number of principal components used to embed the observations in a lower-dimensional space.
+        PCA is run prior to k-nearest-neighbor graph construction.
+  
+    * <p class="parameter">use_thr</p> Boolean, Default: True
+      
+        Specify if you want to use a user-defined threshold in plots to distinguish between true and false doublets.
+        The actual threshold is defined in the following parameter.
+        If set to False, doublet detection will be run with the default threshold calculated by Scrublet.
+        Note that this threshold applies to plots only, meaning no actual filtering takes place here.
+  
+    * <p class="parameter">call_doublets_thr</p> Default: 0.25
+      
+        If use_thr (previous parameter) is set to True, the threshold specified here will be used to define doublets.
+    
+
+### RNA QC    
+This file section specifies parameters used for running Quality Control (QC) on the data.
+Note that we do not filter out cells or genes, as this is part of the subsequent workflow (preprocess).
```

Comment:
This file section specifies parameters used for running Quality Control (QC) on the data. In the ingestion workflow we compute cell and genes QC metrics (such as % mt, ... name a few) but we do not apply filtering, as this is part of the subsequent workflow (preprocess).
Feel free to leave options blank to run with default parameters.